src/script_interface/pair_criteria/pair_criteria.hpp

```diff
@@ -0,0 +1,110 @@
+/*
+  Copyright (C) 2010,2011,2012,2013,2014 The ESPResSo project
+  Copyright (C) 2002,2003,2004,2005,2006,2007,2008,2009,2010
+  Max-Planck-Institute for Polymer Research, Theory Group
+
+  This file is part of ESPResSo.
+
+  ESPResSo is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  ESPResSo is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef SCRIPT_INTERFACE_PAIR_CRITERIA_PAIR_CRITERIA_HPP
+#define SCRIPT_INTERFACE_PAIR_CRITERIA_PAIR_CRITERIA_HPP
+
+#include "../auto_parameters/AutoParameters.hpp"
+#include "core/pair_criteria/pair_criteria.hpp"
+#include <string>
+
+namespace ScriptInterface {
+namespace PairCriteria {
+
+class PairCriterion : public AutoParameters {
```

Comment:
Please remove the data memeber from the base class, and make pair_criterion() pure.
This should then be implemented in the derived classes, and this class should not be exposed to python.
(This is the same solution as with the observables)